src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+                        System.out.println(output);
+                        System.out.println("Now you have 4 tasks in the list.");
+                        arrlst.remove(num - 1);
+                    } else {
+                        System.out.println("siao eh");
+                    }
+                } catch (NumberFormatException nfe) { // add new task like read book, return book
+                    String gotIt = "Got it. I've added this task:";
+
+                    if (arrWords[0].equals("todo")) {
+                        String remainingWords = "";
+                        for (int i = 1; i < arrWords.length; i++) {
+                            remainingWords = remainingWords + " " + arrWords[i];
+                        }
+                        Todo t = new Todo(remainingWords);
+                        arrlst.add(new Task(t.description, t.sym));
+                        System.out.println(gotIt);
+                        System.out.println(t);
+                        String noOfTask = String.format("Now you have %d tasks in the list.", arrlst.size());
+                        System.out.println(noOfTask);
+                    } else if (arrWords[0].equals("deadline")) {
+                        String remainingWords = "";
+                        String dLine = "";
+                        for (int i = 1; i < arrWords.length; i++) {
+                            if (arrWords[i].equals("/by")) {
+                                for (int j = i + 1; j < arrWords.length; j++) {
+                                    dLine = dLine + " " + arrWords[j];
+                                }
+                                break;
+                            } else {
+                                remainingWords = remainingWords + " " + arrWords[i];
+                            }
+                        }
+                        Deadline t = new Deadline(remainingWords, dLine);
+                        arrlst.add(new Task(t.description, t.sym));
+                        System.out.println(gotIt);
+                        System.out.println(t);
+                        String noOfTask = String.format("Now you have %d tasks in the list.", arrlst.size());
+                        System.out.println(noOfTask);
+                    } else if (arrWords[0].equals("event")) {
+                        String remainingWords = "";
+                        String dayAndTime = "";
+                        for (int i = 1; i < arrWords.length; i++) {
+                            if (arrWords[i].equals("/at")) {
+                                for (int j = i + 1; j < arrWords.length; j++) {
+                                    dayAndTime = dayAndTime + " " + arrWords[j];
+                                }
+                                break;
+                            } else {
+                                remainingWords = remainingWords + " " + arrWords[i];
+                            }
+                        }
```

Comment:
Could use comments in places like these to explain what the code is doing (i.e. splitting up the user input) as it may not be apparent on first sight.